modules/likes/jetpack-likes-master-iframe.php

```diff
@@ -4,7 +4,7 @@
  * This function needs to get loaded after the like scripts get added to the page.
  */
 function jetpack_likes_master_iframe() {
-	$version = '20170629';
+	$version = '20171122v2';
```

Comment:
I don't remember running into `v2` appended to date in rest of the codebase. Is there some specific reason for this, and could we just use `20171123` instead?

Reply:
Actually I think we can also use `JETPACK__VERSION` here to avoid the need to bump this manually in the future. Tested it and it seemed to work fine with new translations.

Reply:
> could we just use 20171123 instead?

At this point, sure, we're onto that date. I did `v2` as I seemed to cache a bad variant I tested with under `20171122`.